src/overlays/actors/ovl_En_Sw/z_en_sw.c

```diff
@@ -462,7 +472,7 @@ void func_80B0CF44(EnSw* this, PlayState* play, s32 cnt) {
     }
 }
 
-void func_80B0D14C(EnSw* this, PlayState* play, s32 cnt) {
+void EnSw_SpawnDust2(EnSw* this, PlayState* play, s32 cnt) {
```

Comment:
Usually we use the `funcName2` for identical copies of `funcName` but this isn't 100% identical, I think the dust ring this makes has a bigger radius with `EnSw_SpawnDust2` ?